src/rt/util/container/hashtab.d

```diff
@@ -146,11 +146,10 @@ private:
 
     static hash_t hashOf(in ref Key key) @trusted
     {
-        import rt.util.hash : hashOf;
         static if (is(Key U : U[]))
-            return hashOf(key, 0);
+            return .hashOf(key, 0);
         else
-            return hashOf((&key)[0 .. 1], 0);
+            return .hashOf((&key)[0 .. 1], 0);
```

Comment:
This changes the meaning! Is that intentional? The old version was hashing the key as bytes.

Reply:
I use `object.hashOf` instead of `rt.util.hash : hashOf`, because `rt.util.hash` has been removed.
I may force use `object.hashOf(void[])` if we strongly want bytes hash, but I think it is not needed.